src/Common/src/System/Data/Common/BasicFieldNameLookup.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+//------------------------------------------------------------------------------
```

Comment:
I am not sure why `//------` are around. I have found these from the very beginning and left them around. 

I will clean it up in `BasicFieldNameLookup`